npc/re/jobs/novice/academy.txt

```diff
@@ -13,75 +13,241 @@
 //= 1.0.0 Initial Release. [Kisuka]
 //= 1.0.1 Converted item constants to IDs and added hair coupon. [Kisuka]
 //= 2.0.0 Converted from Hercules Script to rAthena Script. [Capuche]
+//= 2.0.1 Add new Intro Renewal based on Hercules [Jenkijo]
 //==============================================================================
 
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
```

Comment:
No but this might be even more tricky. Need to confirm something first though. Because it looks like the navigation target is being saved when the character logs out. Do not take this for granted though! :)